vizro-core/src/vizro/models/_dashboard.py

```diff
@@ -159,6 +170,21 @@ def _get_page_divs(self, page: Page) -> _PageDivsType:
         return html.Div([dashboard_title, settings, page_title, nav_bar, nav_panel, control_panel, components])
 
     def _arrange_page_divs(self, page_divs: _PageDivsType):
+        collapsable_icon = dmc.Tooltip(
+            html.Span(
+                "keyboard_double_arrow_right",
+                className="material-symbols-outlined",
+                id="collapse_icon",
+            ),
+            id="collapse_tooltip",
+            label="Hide Menu",
+            offset=24,
+            withArrow=True,
+            position="right",
+            arrowOffset=10,
+            className="collapse-button-tooltip",
```

Comment:
Let's use dash notation instead of underscore notation for all the newly added CSS IDs and classNames